### PR TITLE
Fix Module Deprecated -> module

### PR DESCRIPTION
**Pull request by drorta**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/swerve/module/Module.java [hunk 1]
```diff
@@ -160,13 +160,17 @@ public void setSteerVoltage(double voltage) {
 
 	public void pointSteer(Rotation2d steerTargetPosition, boolean optimize) {
 		SwerveModuleState moduleState = new SwerveModuleState(0, steerTargetPosition);
-		targetState.angle = optimize ? targetState.optimize(moduleState, getSteerPosition()).angle : moduleState.angle;
+		if (optimize) {
+			targetState.optimize(getSteerPosition());
```
Comment (hillelv) [inline src/main/java/frc/robot/subsystems/swerve/module/Module.java:164]: don't you mean moduleState.optimize?

File: src/main/java/frc/robot/subsystems/swerve/module/Module.java [hunk 2]
```diff
@@ -160,13 +160,17 @@ public void setSteerVoltage(double voltage) {
...
+		}
+		targetState.angle = moduleState.angle;
 		setTargetSteerPosition(targetState.angle);
 	}
 
 
 	public void setTargetState(SwerveModuleState targetState, boolean isClosedLoop) {
-		this.targetState = SwerveModuleState.optimize(targetState, getSteerPosition());
+		targetState.optimize(getSteerPosition());
+		this.targetState = targetState;
```
Comment (YoniKiriaty) [inline src/main/java/frc/robot/subsystems/swerve/module/Module.java:173]: can be 1 line

Comment (drorta) [reply src/main/java/frc/robot/subsystems/swerve/module/Module.java:173]: How

File: src/main/java/frc/robot/subsystems/swerve/module/Module.java (hunk 1 above)
Comment (drorta) [reply src/main/java/frc/robot/subsystems/swerve/module/Module.java:164]: No such variable, this is the module state

File: src/main/java/frc/robot/subsystems/swerve/module/Module.java (hunk 2 above)
Comment (YHerm) [reply src/main/java/frc/robot/subsystems/swerve/module/Module.java:173]: oh optimize is shit and not returning this

**Review by YHerm — APPROVED** [review]

**Pull request merged**